CT-619 - Changes to account status and audit results pages

danjoneslf opened the pull request:
Includes CT-620

agalamatis commented:
A couple of things about the "/check/{id}/{status}" route.

* It would be better the status string to be explicitly checking for "passed" and "failed" and the else case return 404.

* It would be ideal show which resources are marked with an exception.

danjoneslf commented:
I've gone back to the 2 status tabs model. The reason I hadn't done this before is because the passed / excepted resources aren't being aggregated separately in the audit_criterion record. 

For now I've done pretty much what I showed before so the passed route includes both passed and excepted resources but I've printed the status against each resource so it will have an excepted tag rather than a pass tag. In that case. The "Make an exception" button will behave correctly but I should probably also change the wording.

danjoneslf commented:
Changed the exception button text and updated all the criteria classes to ensure that they reset the annotation to an empty string either at the start of the evaluation method or on a compliant evaluation so we don't get annotations from previous resources.
